docs/components/embedding-models.mdx

```diff
@@ -29,7 +29,7 @@ from embedchain import Pipeline as App
 os.environ['OPENAI_API_KEY'] = 'xxx'
 
 # load embedding model configuration from config.yaml file
-app = App.from_config(yaml_path="config.yaml")
+app = App.from_config(yaml_config="config.yaml")
```

Comment:
This is a breaking change for the user. I think we should still support the `yaml_path` here. 

Going forward, I see the following accepted arguments in this `from_config` function:
- `yaml_path`: path to yaml file
- `config`: dictionary containing the full config
- `json_path`: path to json file

Can you please make this change before we merge this PR?

Reply:
Updated